.env.dist

```diff
@@ -1,70 +1,70 @@
 # App configuration
+#############################
+## Required variables
+#### This value is used to set the sender's address in emails
 SENDER_ADDRESS=noemail@example.com
-ALLOW_FRONT_REDIRECT_PATTERN='/^http[s]*:\/\/(example.com|example2.com)/'
-WEB_CLIENT_URL=
-#Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
-SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
-OAUTH_ALLOWED_DOMAINS=example.com,gmail.com
 
-## Replace with your UID(echo $UID)
-PARENT_USER_ID=1001
-# Docker configuration!
-SERVER_HTTP_PORT=80
+#### Access to admin panel
+##### Gen new pass - bin/console security:encode-password 'pass_here' 'Symfony\Component\Security\Core\User\User'
+SUPERADMIN_PASS='$2y$13$BNuHqfNls76Bsqmb8X8UfOTDWbpTwu0VTxPYw65Um0iDw.KPFYzNe'
 
-# Postgres credentials
+#### Data Base Configuration
 POSTGRES_DB=dev
 POSTGRES_USER=dev
 POSTGRES_PASSWORD=dev
-
-###> doctrine/doctrine-bundle ###
-# Format described at http://docs.doctrine-project.org/projects/doctrine-dbal/en/latest/reference/configuration.html#connecting-using-a-url
-# For an SQLite database, use: "sqlite:///%kernel.project_dir%/var/data.db"
-# Configure your db driver and server_version in config/packages/doctrine.yaml
 DATABASE_URL=pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db:5432/${POSTGRES_DB}
 TEST_DATABASE_URL=pgsql://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db_test:5432/${POSTGRES_DB}
```

Comment:
@mg-4xxi what is it?

Reply:
doctrine settings for db connection
actually we can replace these to doctrine.yaml too